src/components/NewsCard/index.jsx

```diff
@@ -15,35 +14,35 @@ const NewsArticle = ({date, title, article}) => {
     )
 }
 
-const NewsCard = ({source, price, buttonText, details, ...otherProps}) => {
+const NewsCard = ({source, children, ...otherProps}) => {
```

Comment:
The files are not formatted properly 
Please hit cmd + opt + L on an IDE

Reply:
Always a good idea to format before committing. Perhaps you might want to install prettier to have it done automatically.

Reply:
will install prettier and fix this, thanks